File: api_service/api/service.py
from fastapi import APIRouter, Depends
from starlette.responses import Response

from models.status import Status
from services.service import service_service

router = APIRouter()


@router.post('/clear')
async def clear_database(
):
    await service_service.clear_database()

    await service_service.db.close()  # FIXME говнокод

    return Response()


@router.get('/status')
async def get_database_status(
) -> Status:
    status = await service_service.get_status()
    await service_service.db.close()  # FIXME говнокод
    return status

File: api_service/api/service.go
package api

import (
	"db_tp/services"
	"github.com/mailru/easyjson"
	"github.com/valyala/fasthttp"
)

func ClearDatabase(ctx *fasthttp.RequestCtx) {

}

func GetDatabaseStatus(ctx *fasthttp.RequestCtx) {
	status := services.DatabaseSrv.GetStatus()

	resp, _ := easyjson.Marshal(status)
	ctx.Response.SetBody(resp)
	ctx.SetContentType("application/json")
	ctx.Response.SetStatusCode(200)
}
